src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -5955,7 +5955,7 @@ protected final JavadocOptions buildJavadocOptions() throws IOException {
         options.setBootclasspathArtifacts(toList(bootclasspathArtifacts));
         options.setDocfilesSubdirsUsed(docfilessubdirs);
         options.setDocletArtifacts(toList(docletArtifact, docletArtifacts));
-        options.setExcludedDocfilesSubdirs(excludedocfilessubdir);
+        options.setExcludedDocfilesSubdirs(excludedocfilessubdir == null ? null : excludedocfilessubdir.trim());
```

Comment:
This was hard to read. Consider renaming excludedDocfilesSubdirs

Reply:
I don't want to have to rename a Mojo parameter this will be non backward compact. https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#excludedocfilessubdir